meta-aos-vm-common/recipes-aos/aos-deprov/files/deprovision.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+{
+    sleep 1
+
+    # use systemctl stop all aos.target services instead of systemctl stop aos.target, because this approach doesn't wait
+    # all services really stopped.
+    systemctl stop -- $(systemctl show -p Wants aos.target | cut -d= -f2)
+
+    #restore unprovisioned flag
+    rm /var/aos/.provisionstate
```

Comment:
Not sure if it's possible to have no file at this stage, but what if ? :)
```suggestion
    rm /var/aos/.provisionstate -rf
```